Cargo.toml

```diff
@@ -35,7 +34,7 @@ default-features = false
 features = ["use-wasm-bindgen"]
 
 [dev-dependencies]
-mockiato = "0.4"
+mockiato = { git = "https://github.com/myelin-ai/mockiato" }
```

Comment:
Release mockiato and use the correct version

Reply:
Already in a todo